infra/resources/_modules/function_apps_msgs_app/function_app_msgs_app.tf

```diff
@@ -0,0 +1,34 @@
+module "function_app_messages_app" {
+  source = "github.com/pagopa/dx//infra/modules/azure_function_app?ref=main"
+
+  environment = {
+    prefix          = var.prefix
+    env_short       = var.env_short
+    location        = var.location
+    domain          = var.domain
+    app_name        = "app-l${var.index}"
```

Comment:
we should change name of this function, as "app function" doesn't mean anything. Also, `l${var.index}` doesn't add any value here as the `instance_number` field is in charge of creating an enumeration
Any suggestion?

Reply:
these are `msgs-app-func`, `msgs` is domain and `app` means that they serve the app.

Reply:
fixed with https://github.com/pagopa/io-messages/pull/10/commits/87e25d24aebaf97ff0420679910081f0781bd3ea renaming into "citizen"